src/app/projects/pages/projects-list/projects-list.component.ts

```diff
@@ -113,6 +113,7 @@ export class ProjectsListComponent implements OnInit, OnDestroy {
       arrayExpressAccessions: 'Arrayexpress',
       geoAccessions: 'GEO',
       egaStudiesAccessions: 'EGA',
+      dbgapAccessions: 'dbGaP',
```

Comment:
Did you consider reusing the names defined in the ProjectService?

Reply:
I think it's better not to reuse here. The names defined in the ProjectService are allowed data locations and are to do with the UI and may not necessarily be the same as spreadsheet output (in fact they're not, spreadsheet only shows EGAS, not EGAD)